documentation/versioned_docs/version-v0.15/modules/chat/tutorials/start_chatting.mdx

```diff
@@ -0,0 +1,85 @@
+# Chatting
+
+Chat module can be used to add chatting functionality to your application without having to write complex lines of code.
+To begin explaining how to create a chat functionality, some assumptions must be made.
+
+## Basics
+
+Chat communication is only possible between Conduit users. So, first, we must [create some users](../../authentication/get-started#create-a-user).
+We'll need a chat room with some users to participate in now that users have been created. We can accomplish this in two ways:
```

Comment:
~"Having created some users, we can create a chat room in a couple of ways..." or sth